tests/test_testimony.sh

```diff
@@ -0,0 +1,47 @@
+echo "==============="
+echo "= auto report ="
+echo "==============="
+echo
+testimony auto tests
```

Comment:
please use `--nocolor` or `-n` for all testimony calls to disable colored outputs.  This comment is applicable to all references below.